build_tools/benchmarks/post_benchmark_comment.py

```diff
@@ -173,17 +191,45 @@ def main(args: argparse.Namespace):
   if github_token is None:
     raise ValueError("GITHUB_TOKEN must be set.")
 
-  gist_bot_user = os.environ.get("GIST_BOT_USER")
-  if gist_bot_user is None:
-    raise ValueError("GIST_BOT_USER must be set.")
+  comment_bot_user = os.environ.get("COMMENT_BOT_USER")
+  if comment_bot_user is None:
+    raise ValueError("COMMENT_BOT_USER must be set.")
 
   gist_bot_token = os.environ.get("GIST_BOT_TOKEN")
   if gist_bot_token is None:
     raise ValueError("GIST_BOT_TOKEN must be set.")
 
-  pr_number = args.pr_number
   comment_data = benchmark_comment.CommentData(
       **json.loads(args.comment_json.read_text()))
+  # Sanitize the pr number to make sure it is an integer.
+  pr_number = int(comment_data.unverified_pr_number)
+
+  if args.no_verify_pr:
+    github_event = None
+  else:
+    github_event = json.loads(args.github_event_json.read_text())
+
+  pr_client = GithubClient(requester=APIRequester(github_token=github_token))
+  if github_event is not None:
```

Comment:
Seems like this should be combined with the if condition above?

```suggestion
  pr_client = GithubClient(requester=APIRequester(github_token=github_token))
  if args.github_event:
    github_event = json.loads(args.github_event_json.read_text())
```

Reply:
Done